var/spack/repos/builtin/packages/wgrib2/package.py

```diff
@@ -3,29 +3,204 @@
+
+    variant_map = {
+        "netcdf3": "USE_NETCDF3",
+        "netcdf4": "USE_NETCDF4",
+        "spectral": "USE_SPECTRAL",
+        "mysql": "USE_MYSQL",
+        "udf": "USE_UDF",
+        "regex": "USE_REGEX",
+        "tigge": "USE_TIGGE",
+        "proj4": "USE_PROJ4",
+        "aec": "USE_AEC",
+        "g2c": "USE_G2CLIB",
+        "png": "USE_PNG",
+        "jasper": "USE_JASPER",
+        "openmp": "USE_OPENMP",
+        "wmo_validation": "USE_WMO_VALIDATION",
+        "ipolates": "USE_IPOLATES",
+        "disable_timezone": "DISABLE_TIMEZONE",
+        "disable_alarm": "DISABLE_ALARM",
+        "fortran_api": "MAKE_FTN_API",
+    }
+
+    # Disable parallel build
+    parallel = False
+
+    # Use Spack compiler wrapper flags
+    def inject_flags(self, name, flags):
+        if name == "cflags":
+            if self.spec.compiler.name == "apple-clang":
+                flags.append("-Wno-error=implicit-function-declaration")
+
+            # When mixing Clang/gfortran need to link to -lgfortran
+            # Find this by searching for gfortran/../lib
+            if self.spec.compiler.name in ["apple-clang", "clang"]:
+                if "gfortran" in self.compiler.fc:
+                    output = Executable(self.compiler.fc)("-###", output=str, error=str)
+                    libdir = re.search("--libdir=(.+?) ", output).group(1)
+                    flags.append("-L{}".format(libdir))
+
+        return (flags, None, None)
+
+    flag_handler = inject_flags
+
+    def url_for_version(self, version):
+        url = "https://www.ftp.cpc.ncep.noaa.gov/wd51we/wgrib2/wgrib2.tgz.v{}"
+        return url.format(version)
+
+    def edit(self, spec, prefix):
+        makefile = FileFilter("makefile")
+
+        # ifort no longer accepts -openmp
+        makefile.filter(r"-openmp", "-qopenmp")
+        makefile.filter(r"-Wall", " ")
+        makefile.filter(r"-Werror=format-security", " ")
+
+        # clang doesn"t understand --fast-math
+        if spec.satisfies("%clang") or spec.satisfies("%apple-clang"):
+            makefile.filter(r"--fast-math", "-ffast-math")
+
+        for variant_name, makefile_option in self.variant_map.items():
+            value = int(spec.variants[variant_name].value)
+            makefile.filter(r"^%s=.*" % makefile_option, "{}={}".format(makefile_option, value))
+
+    def setup_build_environment(self, env):
+
+        if self.spec.compiler.name in "intel":
+            comp_sys = "intel_linux"
+        elif self.spec.compiler.name in ["gcc", "clang", "apple-clang"]:
+            comp_sys = "gnu_linux"
+
+        env.set("COMP_SYS", comp_sys)
+
+    def build(self, spec, prefix):
+
+        # Get source files for netCDF4 builds
+        if self.spec.satisfies("+netcdf4"):
+            with working_dir(self.build_directory):
+                os.system(
+                    "wget https://downloads.unidata.ucar.edu/netcdf-c/4.8.1/netcdf-c-4.8.1.tar.gz"
```

Comment:
Same here, but that is really not something I should be doing. That's up to the package creators at EMC.

Reply:
@t-brown Can we get a motion on this PR? I am only the messenger trying to update the package in the authoritative spack repo (and fixing bugs along the way). As @edwardhartnett said, people are encouraged to move away from `wgrib2` since its author has refused to cooperate on improving the package.

Reply:
I thought you were removing me as a maintainer. If so, I should not be holding you up at all.